site/agents-mcp/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@cloudflare/agents-mcp",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "wrangler dev",
+    "deploy": "wrangler deploy",
+    "types": "wrangler types"
+  },
+  "dependencies": {
+    "@cfworker/json-schema": "^4.1.1",
```

Comment:
this won't be necessary if we land https://github.com/cloudflare/agents/pull/620/files#diff-0b2ee78feb201d3caa4914dcbf17821b7a21d6e5d0f708fc2eadcf7ee8f83c38R28